index.html

```diff
@@ -29,29 +35,25 @@ <h2 class="title">Memory Game</h2>
             <input type="radio" name="language" value="danish" />Danish
           </label>
         </div>
-        <label for="category">
-          Choose a category
-          <select id="category" name="category-list">
-            <option value="animals">Animals</option>
-            <option value="weather">Weather</option>
-            <option value="clothes">Clothes</option>
-          </select>
-          <br />
-        </label>
-        <label for="user-timer">
-          Set a time limit:
-          <input
-            id="user-timer"
-            type="number"
-            name="user-timer"
-            min="10"
-            max="120"
-            value="15"
-          />
-        </label>
+        <label for="category"> Choose a category</label>
+        <select id="category" name="category-list">
+          <option value="animals">Animals</option>
+          <option value="weather">Weather</option>
+          <option value="clothes">Clothes</option>
+        </select>
+        <label for="user-timer"> Set a time limit in seconds: </label>
+        <input
+          id="user-timer"
+          type="number"
+          name="user-timer"
+          min="20"
```

Comment:
⚠️ I tested by entering less than 20 seconds and more than 120, and the restriction is not applied. 
💡My suggestion, if you want to set a time range, is to implement some validation using JS (perhaps with an IF block).

Reply:
I'm glad you tried to break it, Nico!